modules/cats/src/test/scala/pureconfig/module/cats/CatsSuite.scala

```diff
@@ -0,0 +1,23 @@
+package pureconfig.module.cats
+
+import cats.data.{ NonEmptyList, NonEmptyVector }
+import com.typesafe.config.ConfigFactory
+import org.scalatest._
+import pureconfig.syntax._
+
+class CatsSuite extends FlatSpec with Matchers with EitherValues {
+
+  case class Numbers(numbers: NonEmptyList[Int])
+
+  it should "be able to read a config with a NonEmptyList" in {
+    val config = ConfigFactory.parseString(s"""{ numbers: [1,2,3] }""")
+    config.to[Numbers].right.value shouldEqual Numbers(NonEmptyList(1, List(2, 3)))
```

Comment:
It actually doesn't throw an exception, since it's using ScalaTest's `EitherValues` - you get an error message like, "The Either on which right.value was invoked was not defined as a Right." But I'll change it.

FYI, we also use `EitherValues` in many other test suites in core and modules, so if this is something we're moving away from, we should file an issue.